optimum/neuron/utils/hub_neuronx_cache.py

```diff
@@ -201,24 +201,47 @@ def _create_hub_compile_cache_proxy(
     return CompileCacheHfProxy(cache_repo_id, default_cache, endpoint=endpoint, token=token)
 
 
-@dataclass
-class CacheEntry:
-    key: str
-    metadata: Dict[str, Any]
+class ModelCacheEntry:
```

Comment:
Awesome! I will leverage the class for encoder models's caching as well, thx David!